web-calculator.js

```diff
@@ -0,0 +1,25 @@
+var a, b, result;
+function setValues() {
+    a = Number(document.getElementById('a').value);
+    b = Number(document.getElementById('b').value);
+}
+function ADD() {
```

Comment:
JavaScript prefers camelCase that means all lower case except when it is the combination of words, then the first letter of each word starting with the second is capitalized. Like `getElementById`